### ISSUE TITLE
Toggl

### ISSUE THREAD

**Issue by aridizSTtest** [bitfocus/companion-module-requests#629]
I know AV Production is a large percentage of where this project is going...
But if you wanted to open an additional avenue, time tracking would be very useful.
Toggle is a leading service.
With an open API with good documentation.

Also, Feel free to research other services like Pomodoro Timers for more information on Time Management Services.

Thanks

**Comment by josephdadams** [comment]
Feel free to include links to the open API with good documentation

**Comment by daniep01** [comment]
Here's the API docs
https://github.com/toggl/toggl_api_docs

If there are no other takers I might make this module

**Comment by josephdadams** [comment]
@daniep01 it's yours

**Comment by daniep01** [comment]
@josephdadams 

Can I just pull in this node.js library and use it in my module?
https://github.com/7eggs/node-toggl-api

**Comment by josephdadams** [comment]
Yep, just do a `yarn add toggl-api` in your module's folder in the `module-local-dev`, it will add it as a dependency and put it in your `package.json`

**Comment by daniep01** [comment]
This module has been created and will be in the next beta of Companion
